docs/setup.md

```diff
@@ -173,6 +184,22 @@ When you access the `profile.` subdomain, a `GU_GATEWAY` cookie is automatically
 
 Once this cookie value is `true`, you'll automatically be directed to the `gateway` routes that have been migrated so far.
 
+## Storybook
+
+You can use [Storybook](https://storybook.js.org/)to build UI components and pages in isolation without having to launch the Gateway server or set up routes.
+
+Simply run the following command to start Storybook locally
+
+```sh
+$ yarn story
+```
+
+There's more documentation on setting up a story in the [development documentation](development.md).
+
+When a git branch is updated on GitHub, a "Chromatic Deploy" action is run which publishes the Storybook to a URL.
+
+It also runs visual regression tests, and the "UI Tests" must pass/be accepted before the PR can be merged. This is used to prevent unintended UI changes from making their way into production.
```

Comment:
```suggestion
It also runs [visual regression tests](https://www.chromatic.com/), and the "UI Tests"§ must pass/be accepted before the PR can be merged. This is used to prevent unintended UI changes from making their way into production.
```